schemas/ethereum.graphql

```diff
@@ -0,0 +1,86 @@
+scalar String
```

Comment:
What do you think about having the manifest schemas inside each of their respective `protocol` folder?

Like this:
```
src
│   ...
└─── protocols
   └─── ethereum
      └─── manifest.graphql
   └─── near
      └─── manifest.graphql

Reply:
I thought about it because I saw the `protocol` subfolder. If you prefer that location, I'm all good with it. Ask and you shall receive :D